packages/rest/src/rest.server.ts

```diff
@@ -190,6 +192,25 @@ export class RestServer extends Context implements Server, HttpServerLike {
     this._setupRequestHandler();
 
     this.bind(RestBindings.HANDLER).toDynamicValue(() => this.httpHandler);
+
+    // LB4's static assets serving router
+    const staticAssetsRouter = new StaticRoute(
+      (req: Request, res: Response) => {
+        return new Promise((resolve, reject) => {
+          const onFinished = () => resolve();
+          res.once('finish', onFinished);
+          this._routerForStaticAssets.handle(req, res, (err: Error) => {
+            if (err) {
+              return reject(err);
+            }
+            // Express router called next, which means no route was matched
+            return reject(new HttpErrors.NotFound());
+          });
+        });
+      },
+    );
+
+    this.httpHandler.registerRoute(staticAssetsRouter);
```

Comment:
IIUC, this is relying on the fact that the code paths invoked here is bypassing path validations and that our routing table will pass the path `*` to Express router without modifications. 

I find that rather brittle, let's find a different solution please.

I am thinking about the following implementation of [`RoutingTable#find`](https://github.com/strongloop/loopback-next/blob/a2ab86d3a3c721f28159585ca4e87bd8504476c4/packages/rest/src/router/routing-table.ts#L173-L188):

```ts
class RoutingTable {
  // ...

  find(request: Request): ResolvedRoute {
    debug('Finding route %s for %s %s', request.method, request.path);

    const found = this._router.find(request);

    if (found) {
      debug('Route matched: %j', found);
      return found;
    }

    debug('No API route found for %s %s, trying to find a static assets', request.method, request.path);
    return this._staticAssetsRoute;
  }
}
```